doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_suit_hierarchical_manifests.rst

```diff
@@ -8,7 +8,7 @@ Hierarchical manifests
    :depth: 2
 
 The Software Updates for the Internet of Things (SUIT) procedure provides the option to employ a hierarchical manifest system.
-In theory the SUIT SFU procedure could be executed with a single manifest however, this approach could lead to certain limitations.
+In theory, the SUIT DFU procedure could be executed with a single manifest; however, this approach would lead to certain limitations.
```

Comment:
```suggestion
The SUIT DFU procedure could be executed with a single manifest; however, this approach would lead to certain limitations as multiple manifests offer more tailored-control of the DFU process.
```